tests/acceptance/features/apiSpacesShares/copySpaces.feature

```diff
@@ -228,19 +231,23 @@ Feature: copy file
     And for user "Alice" the content of the file "/testshare/personal.txt" of the space "Shares" should be "personal content"
 
 
-  Scenario: user copies a file from personal space to share space with role viewer
+  Scenario Outline: user copies a file from personal space to share space with role viewer and secure viewer
     Given user "Brian" has created folder "/testshare"
     And user "Brian" has sent the following resource share invitation:
-      | resource        | testshare |
-      | space           | Personal  |
-      | sharee          | Alice     |
-      | shareType       | user      |
-      | permissionsRole | Viewer    |
+      | resource        | testshare          |
+      | space           | Personal           |
+      | sharee          | Alice              |
+      | shareType       | user               |
+      | permissionsRole | <permissions-role> |
     And user "Alice" has uploaded file with content "personal content" to "/personal.txt"
     When user "Alice" copies file "/personal.txt" from space "Personal" to "/testshare/personal.txt" inside space "Shares" using the WebDAV API
```

Comment:
IMO, we don't have to test with secure-view here. we want to test the reverse (from Secure-view share to Personal space)